### ISSUE TITLE
Product Area: Grafana OSS (`grafana-server`) v10.1.1 crashes upon startup on armhf after upgrade from v10.0.3

### ISSUE THREAD

**Issue by vincejv** [grafana/grafana#74611]
### What happened?

Grafana OSS (`grafana-server`) v10.1.1 crashes upon startup on armhf (Orange PI One) after upgrade from v10.0.3

The workaround is to downgrade to v10.0.3 and prevent auto upgrade using the following commands
```
sudo apt-mark hold grafana
sudo apt-get install grafana=10.0.3
```

Not sure if relevant, but a similar arm-specific issue was raised last year, the same issue that `grafana-server` crashes upon startup but a different error message this time: https://github.com/grafana/grafana/issues/48130

### What did you expect to happen?

Grafana server to start successfully without fatal errors

### Did this work before?

Yes, Grafana server instance started without any issues on 10.0.3

### How do we reproduce it?

1. Run `sudo systemctl start grafana-server.service`
2. Run `systemctl status grafana-server.service`
3. Greeted with failed service with the following logs
```
● grafana-server.service - Grafana instance
     Loaded: loaded (/lib/systemd/system/grafana-server.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Fri 2023-09-08 21:24:11 PST; 1h 28min ago
       Docs: http://docs.grafana.org
    Process: 2032 ExecStart=/usr/share/grafana/bin/grafana server --config=${CONF_FILE} --pidfile=${PID_FILE_DIR}/grafana-server.pid --packaging=de>
   Main PID: 2032 (code=exited, status=127)
```

```golang
-- Logs begin at Fri 2023-09-08 20:17:04 PST. --
Sep 08 22:53:19 mithras grafana[6754]: Error relocating /usr/share/grafana/bin/grafana: __stat_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6754]: Error relocating /usr/share/grafana/bin/grafana: __time64: symbol not found
Sep 08 22:53:19 mithras grafana[6754]: Error relocating /usr/share/grafana/bin/grafana: __utimes_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6754]: Error relocating /usr/share/grafana/bin/grafana: __dlsym_time64: symbol not found
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Main process exited, code=exited, status=127/n/a
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Failed with result 'exit-code'.
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Scheduled restart job, restart counter is at 4.
Sep 08 22:53:19 mithras systemd[1]: Stopped Grafana instance.
Sep 08 22:53:19 mithras systemd[1]: Started Grafana instance.
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __lstat_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __localtime64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __gettimeofday_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __nanosleep_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __fstat_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __stat_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __utimes_time64: symbol not found
Sep 08 22:53:19 mithras grafana[6757]: Error relocating /usr/share/grafana/bin/grafana: __dlsym_time64: symbol not found
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Main process exited, code=exited, status=127/n/a
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Failed with result 'exit-code'.
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Scheduled restart job, restart counter is at 5.
Sep 08 22:53:19 mithras systemd[1]: Stopped Grafana instance.
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Start request repeated too quickly.
Sep 08 22:53:19 mithras systemd[1]: grafana-server.service: Failed with result 'exit-code'.
Sep 08 22:53:19 mithras systemd[1]: Failed to start Grafana instance.
```

### Is the bug inside a dashboard panel?

No

### Environment (with versions)?

Grafana: 10.0.11
OS: Ubuntu 20.04.6 LTS (Focal Fossa), Armbian 23.02.2 commit 1a8daf0
Browser: n/a
System: Orange PI One with the following info on `/cat/proc/cpuinfo`
```
processor       : 0
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 22.85
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 1
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 22.85
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 2
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 22.85
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

processor       : 3
model name      : ARMv7 Processor rev 5 (v7l)
BogoMIPS        : 22.85
Features        : half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xc07
CPU revision    : 5

Hardware        : Allwinner sun8i Family
Revision        : 0000
Serial          : 02c00081d3dc192c
```

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

### Datasource(s)?

Telegraf, Prometheus

**Comment by bitsurfr** [comment]
The same issue is happening to me and I can confirm downgrading to 10.0.3 resolves the problem.

**Comment by vincejv** [comment]
Similar issue raised 3 years ago https://github.com/grafana/grafana/issues/23500

**Comment by NSBrianWard** [comment]
I'm holding at 10.0.5 right for the moment and that's working for me.

**Comment by NSBrianWard** [comment]
Here's my system info just in case it's helpful (output of ``cat /proc/cpuinfo``):

```
processor	: 0
model name	: ARMv7 Processor rev 4 (v7l)
BogoMIPS	: 38.40
Features	: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xd03
CPU revision	: 4

processor	: 1
model name	: ARMv7 Processor rev 4 (v7l)
BogoMIPS	: 38.40
Features	: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xd03
CPU revision	: 4

processor	: 2
model name	: ARMv7 Processor rev 4 (v7l)
BogoMIPS	: 38.40
Features	: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xd03
CPU revision	: 4

processor	: 3
model name	: ARMv7 Processor rev 4 (v7l)
BogoMIPS	: 38.40
Features	: half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32 lpae evtstrm crc32
CPU implementer	: 0x41
CPU architecture: 7
CPU variant	: 0x0
CPU part	: 0xd03
CPU revision	: 4

Hardware	: BCM2835
Revision	: a22082
Serial		: 00000000d89cf3ac
Model		: Raspberry Pi 3 Model B Rev 1.2
```

**Comment by nikimanoledaki** [comment]
Hi folks! Given the error log `stat_time64: symbol not found` it seems that a library is missing or might not be available for this arch. This may be related to `musl` which was recently added for Ubuntu builds 2 weeks ago in this PR by @kminehart
 https://github.com/grafana/grafana/pull/74127

**Comment by kminehart** [comment]
Yeah I've seen reports of some older Debian versions don't have a high enough version of musl, though our latest release was statically compiled to include musl so it shouldn't matter what's installed on your machine.

I think we've only shared a deb installer of the latest release the linked thread, but our next public release in ~2 weeks should include the statically included musl libc.

**Comment by vincejv** [comment]
hello all, seems to have been fixed on 10.1.2, closing this issue. Thanks a lot!
